Makefile

```diff
@@ -238,6 +238,7 @@ deps: $(PLUGIN_DIR) $(GO_ETHEREUM_DIR) $(SSHA3_DIR)
 	cd $(HASHICORP_DIR) && git checkout $(HASHICORP_GIT_REV)
 	cd $(BTCD_DIR) && git checkout $(BTCD_GIT_REV)
 	cd $(YUBIHSM_DIR) && git checkout master && git pull && git checkout $(YUBIHSM_REV)
+	cd $(TRANSFER_GATEWAY_DIR) && git checkout master && git pull && git checkout $(TG_GIT_REV)
```

Comment:
The only thing that should change in the Makefile are `GO_LOOM_GIT_REV` and `TG_GIT_REV`, revert all the other changes. The build is failing because `GatewayMigration` function needs the `gateway` build flag, but we need to be able to build the loom binary without the gateway features, so to fix this add another file called `gateway_dummy_migration.go` with `// +build !gateway` in it, and an empty implementation of the `GatewayMigration` function.